web/src/app/tree/trees-list/trees-list.component.ts

```diff
@@ -15,8 +15,12 @@ export class TreesListComponent implements OnInit {
   ) { }
 
   ngOnInit(): void {
+    this.onModelChange();
+  }
+
+  onModelChange() {
     this.service.getMyTrees().subscribe(result => {
```

Comment:
I assume that after creating the tree, you are asking API for list?

Reply:
Exactly, parent component receives event from the child and calls this method

Reply:
U can just append the list with the newly created name. Whatever